tools/elfdeps.c

```diff
@@ -33,6 +38,97 @@ typedef struct elfInfo_s {
     ARGV_t provides;
 } elfInfo;
 
+/*
+ * If filename contains ".so" followed by a version number, return
+ * a copy of the version number.
+ */
+static char *getLibtoolVer(const char *filename)
+{
+    const char *so;
+    char dest[PATH_MAX];
+    int destsize = 0;
+    int found_digit = 0, found_dot = 0;
+
+    destsize = readlink(filename, dest, PATH_MAX);
+    if (destsize > 0) {
+	dest[destsize] = 0;
+	filename = dest;
+    }
+    // Start from the end of the string.  Verify that it ends with
+    // numbers and dots, preceded by ".so.".
```

Comment:
On a purely stylistical nit, `//` comments do not belong in rpm coding style. Always use `/* ... */` and for multiline comments,
```
/*
 * ...
 * ...
 */
```

/me realizes this isn't even explained in contributing.md, sigh...

Reply:
I'll follow up on each of those items tomorrow.  Thanks for the feedback!